src/libraries/Common/src/Microsoft/Win32/SafeHandles/SafeLibraryHandle.cs

```diff
@@ -1,12 +1,19 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
+
 namespace Microsoft.Win32.SafeHandles
 {
     internal sealed class SafeLibraryHandle : SafeHandleZeroOrMinusOneIsInvalid
     {
         public SafeLibraryHandle() : base(true) { }
 
+        internal SafeLibraryHandle(IntPtr value) : base(true)
+        {
+            SetHandle(value);
+        }
+
         protected override bool ReleaseHandle()
         {
             return Interop.Kernel32.FreeLibrary(handle);
```

Comment:
Should we consider changing this to be `NativeLibrary.Free` instead and make it platform-agnostic?

Reply:
Looks like this is used in a couple places that build for versions where `NativeLibrary` doesn't exist, so I left it `FreeLibrary` for now, since it doesn't have a non-Windows use case yet.